src/base/registrations/registrar.ts

```diff
@@ -1,7 +1,7 @@
 import { ethers } from 'ethers';
 import { writable } from 'svelte/store';
 import type { BigNumber } from 'ethers';
-import type { EnsResolver, Resolver } from '@ethersproject/providers';
```

Comment:
What's the difference?

Reply:
There isn't a big difference, but since they are pretty similar we are having places where we use `Resolver` and places where it's `EnsResolver`.
So I took as reference.
https://github.com/radicle-dev/radicle-interface/blob/1fb59faf308d91ad2c77ea35b8a1129e36d838c7/src/base/registrations/registrar.ts#L14
And unified everything under `EnsResolver` since it's name is more explicit.

`EnsResolver` is the interface that Resolver implements and extends.
https://github.com/ethers-io/ethers.js/blob/d3b7130ed6ec50b192eb7f33905eaa327d65eee2/packages/providers/lib/base-provider.d.ts#L31

But I'm seeing now that there a different ethers.js functions that return `Resolver` so eventually we should go back to `Resolver` and unify it everywhere. What do you think?

Reply:
I see, I think using the interface is probably better. It doesn't really matter what the return value is, as it'll match EnsResolver.